### ISSUE TITLE
Getting a "TypeError: undefined is not an object" when running aws-azure-login

### ISSUE THREAD

**Issue by husmith** [aws-azure-login/aws-azure-login#14]
After configuring my aws creds and trying to login, I keep getting the following after entering my password:

```
info: TypeError: undefined is not an object (evaluating 'document.forms[0].login.value = username')
info:   undefined:2 in anonymous
info:   :6
```

When I run `aws-azure-login` with DEBUG turned on, it gives the same "info: TypeError" but then says the page is loaded. I am never given the role.

**Comment by dtjohnson** [comment]
It's a known issue caused by Microsoft updating the login page. Update to the latest version and it should work. It's just a temp fix, but I'm working on a longer term solution.

**Comment by dtjohnson** [comment]
I just published v1.0.0 with a long term fix for the new login page.
